com.icons8.Lunacy.yml

```diff
@@ -0,0 +1,76 @@
+app-id: com.icons8.Lunacy
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: lunacy
+build-options:
+  no-debuginfo: true
+tags:
+  - proprietary
+finish-args:
+  - --persist=.local/share/Icons8/Lunacy
+  - --device=dri
+  - --share=ipc
+  - --socket=cups
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=xdg-config/fontconfig:ro
+  - --talk-name=org.gtk.vfs.*
+  # Network needed for remote asset importing
+  - --share=network
+
+modules:
+  - name: lunacy
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/bin
+      - install -D lunacy ${FLATPAK_DEST}/bin/lunacy
+      - install -D /usr/bin/ar -t ${FLATPAK_DEST}/bin
+      - mkdir -p ${FLATPAK_DEST}/lib
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so ${FLATPAK_DEST}/lib
+      - install -D apply_extra ${FLATPAK_DEST}/bin/apply_extra
+      - install -D -t ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps com.icons8.Lunacy.svg
+      - install -D -t ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps com.icons8.Lunacy.png
+      - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo com.icons8.Lunacy.metainfo.xml
+
+    sources:
+      - type: extra-data
+        filename: Lunacy.deb
+        url: https://lun-eu.icons8.com/s/setup/Lunacy_8.2.1.deb
+        sha256: a66dd7cf50c71490b96d3f1d6e2c88da79a24c6810fb7052466a08decf8202d7
+        size: 206367108
+        only-arches:
+          - x86_64
+        x-checker-data:
+          type: rotating-url
+          url: https://docs.icons8.com/release-notes/
+          pattern: https://lun-eu.icons8.com/s/setup/Lunacy_([0-9.]+).deb
+
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - ar x Lunacy.deb
+          - tar xf data.tar.xz --no-same-owner
+          - mv opt/icons8/lunacy/* .
+          - mkdir -p export/share/applications
+          - sed 's/Icon=.*/Icon=com.icons8.Lunacy/' usr/share/applications/lunacy.desktop | sed 's/Exec=.*/Exec=\/app\/bin\/lunacy %u/' > export/share/applications/com.icons8.Lunacy.desktop
+          - rm -rf control.tar.xz data.tar.xz debian-binary Lunacy.deb opt
+
+      - type: file
+        path: lunacy
+
+      - type: file
+        path: com.icons8.Lunacy.metainfo.xml
+
+      # Icon is from https://icons8.com/vue-static/landings/lunacy-new/lunacy.svg
+      - type: file
+        path: com.icons8.Lunacy.svg
+
+      - type: file
+        path: com.icons8.Lunacy.png
```

Comment:
file is missing. But it shouldn't be necessary with the .svg.

Reply:
That's likely because it is missing the desktop file that appstream-compose expect to be able to copy the icons.

A svg is enough